integration/taskcluster-hook.json

```diff
@@ -0,0 +1,59 @@
+{
+    "bindings": [],
+    "metadata": {
+        "description": "Run code-review integration tests once every day",
+        "emailOnError": true,
+        "name": "Code review hook (CHANNEL)",
+        "owner": "babadie@mozilla.com"
+    },
+    "schedule": [
+        "0 0 7 * * *"
+    ],
+    "task": {
+        "created": {
+            "$fromNow": "0 seconds"
+        },
+        "deadline": {
+            "$fromNow": "2 hours"
+        },
+        "expires": {
+            "$fromNow": "1 month"
+        },
+        "extra": {},
+        "metadata": {
+            "description": "Integration tests for the code-review bot workflow",
+            "name": "Code review integration tests (CHANNEL)",
+            "owner": "babadie@mozilla.com",
+            "source": "https://github.com/mozilla/code-review"
+        },
+        "payload": {
+            "artifacts": {},
+            "cache": {},
```

Comment:
Maybe we should use a cache for the clone

Reply:
Requested clones https://github.com/mozilla/community-tc-config/pull/155

Reply:
They are setup :tada: